.github/workflows/haskell.yml

```diff
@@ -36,7 +36,7 @@ jobs:
       fail-fast: false
       matrix:
         # If you edit these versions, make sure the version in the lonely macos-latest job below is updated accordingly
-        ghc: ["9.6", "9.8"]
+        ghc: ["9.6", "9.8", "9.10"]
```

Comment:
Should we take this opportunity to remove 9.8 as suggested in Slack?

Reply:
Done/